src/Engines/QueryBuilderEngine.php

```diff
@@ -378,6 +378,34 @@ protected function compileQuerySearch($query, $column, $keyword, $relation = 'or
         $column = $this->castColumn($column);
         $sql    = $column . ' LIKE ?';
 
+        /**
+         * Patch for fix about ambiguous field
+         *
+         * Ambiguous field error will appear
+         * when query use join table and search with keyword.
+         */
+        // Remove backtick that appear from MYSQL query.
+        $column = str_replace('`', '', $column);
```

Comment:
I just test on MYSQL only so I found ` only.

I don't know for other DB but I'm sure that `$column` has delimiter for column. I need to remove it before add table name.

After this I need type in code because github markdown and backtick conflict.

```
Example $column is `field`. If I don't remove backtick and add table name.
$column will be table_name.`field` and it will go in wrap method.
Result will look like `table_name`.``field``.

It will error.
```

I don't know which solution to remove delimiter from column name. Do you have any suggestion?